extension/src/plots/data/index.ts

```diff
@@ -72,4 +71,15 @@ export class PlotsData extends BaseData<{ data: PlotsOutput; revs: string[] }> {
   public setModel(model: PlotsModel) {
     this.model = model
   }
+
+  private getArgs(revs: string[]) {
+    if (
+      this.model &&
+      (sameContents(revs, ['workspace']) || sameContents(revs, []))
+    ) {
+      return this.model.getDefaultRevs()
```

Comment:
[F] We can no longer call the CLI with no arguments because the default behaviour is to return the `workspace` & branch name as the revisions. The rest of the code will then fail.